app/assets/stylesheets/components/_reorderable-list.scss

```diff
@@ -0,0 +1,128 @@
+.app-c-reorderable-list {
+  @include govuk-font(19, bold);
+
+  list-style-type: decimal;
+  margin-bottom: govuk-spacing(6);
+  margin-top: 0;
+  padding-left: govuk-spacing(4);
+  position: relative;
+
+  .govuk-form-group {
+    margin-bottom: 0;
+  }
+}
+
+.app-c-reorderable-list__item {
+  margin-bottom: govuk-spacing(3);
+
+  &:after {
+    background: $govuk-border-colour;
+    content: '';
+    display: block;
+    height: 1px;
+    left: 0;
+    position: absolute;
+    width: 100%;
+  }
+
+  @include govuk-media-query($from: desktop) {
+    cursor: move;
+  }
+}
+
+.app-c-reorderable-list__item--chosen {
+  background-color: govuk-colour('light-grey');
+  outline: 2px dotted $govuk-border-colour;
+
+  &:after {
+    content: none;
+  }
+}
+
+.app-c-reorderable-list__item--drag {
+  background-color: govuk-colour('white');
+  padding: govuk-spacing(2);
+
+  &:after {
+    content: none;
+  }
+}
+
+.app-c-reorderable-list__wrapper {
+  display: block;
+
+  @include govuk-media-query($from: desktop) {
+    display: inline-flex;
+    width: 100%;
+  }
+}
+
+.app-c-reorderable-list__content,
+.app-c-reorderable-list__actions {
+  margin-bottom: govuk-spacing(2);
+}
+
+.app-c-reorderable-list__content {
+  @include govuk-media-query($from: desktop) {
+    flex: 0 1 auto;
+    min-width: 70%;
+  }
+}
+
+.app-c-reorderable-list__title {
+  margin: 0;
+}
+
+.app-c-reorderable-list__description {
+  @include govuk-font(16, regular);
```

Comment:
This confused me slightly, because we're making a 'global' font setting and then undoing it here. Would it be simpler to style the title and indices explicitly instead?

Reply:
The global font setting is at the list level and it's required to style the numbers – it also implicitly applies to list items. We're using this as an exception for the description indeed.